setup.py

```diff
@@ -284,7 +284,7 @@ def run(self):
 #   https://packaging.python.org/discussions/install-requires-vs-requirements/
 #
 docs_testenv = linux_testenv + distributed_env + [
-    'sphinx',
+    'sphinx <= 2.4.4',  # avoid `sphinx >= 3.0` that breaks build
```

Comment:
Why? Can you link to the specific issue here?

Reply:
No idea, just don't build with latest sphinx

Reply:
Separate issue created (thanks to @mpenkov) https://github.com/RaRe-Technologies/gensim/issues/2793